expirationd.lua

```diff
@@ -265,6 +269,16 @@ local function default_tuple_drop(space_id, args, tuple, task)
 end
 
 
+-- default iterate_with function
+local function default_iterate_with(task)
+    return task.index:pairs({ task.start_key() }, { iterator = task.iterator_type })
+               :take_while(
+            function(tuple)
+                return task:process_while()
+            end
+    )
```

Comment:
Probably someone will argue me but this indents look ugly.

Reply:
I think the new version should be more like it